### DISCUSSION TITLE
Definitely a bug, but one you should NEVER fix!

### DISCUSSION THREAD

**Discussion by HetDerwel** [nlohmann/json#2227] [General]
Hi Nlohmann,

I have to say this and raise it as a personal bug and I really hope everyone doesn't mind, because I've rarely if EVER found a developer so happy to help and answer questions.  Not only have you developed a fantastic library, but the questions you've already answered have provided myself with many of the answers I've been looking for.

May you continue to provide such excellent help to all that come looking and so many other developers could do with taking a leaf out of your book.  Maybe I'm wrong and this isn't a bug after-all, but more of a feature.

I'm just astounded.  Thank you.

**Comment by nlohmann** [comment]
Thanks!

**Reply by HetDerwel** [reply]
I just wanted to make sure that everyone knows how much I personally appreciate your hard work and excellent library.  In fact, I've convinced the whole coding team to use it.  It's SO much better than that jansson thing we were using.  We're all much happier.  So
once again, thank you.

I hope nobody minded me raising it as an issue though!  hehehe.
